ArduCopter/GCS_Mavlink.cpp

```diff
@@ -570,6 +570,48 @@ bool GCS_MAVLINK_Copter::set_home(const Location& loc, bool _lock) {
     return copter.set_home(loc, _lock);
 }
 
+MAV_RESULT GCS_MAVLINK_Copter::handle_command_int_do_reposition(const mavlink_command_int_t &packet) {
```

Comment:
nitpick: the curly bracket should be on the next line.

Reply:
Done. I was happy the previous 2 functions were on the same line so I just copied that :) I generally just try and mimic whatever the style for a given file already was, but I see I didn't look far enough.